src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenAsyncMainTests.cs

```diff
@@ -725,6 +725,45 @@ string getMethodName(MethodDefinitionHandle handle)
             IEnumerable<string> getMethodAttributes(MethodDefinitionHandle handle)
                 => metadataReader.GetMethodDefinition(handle).GetCustomAttributes()
                     .Select(a => metadataReader.Dump(metadataReader.GetCustomAttribute(a).Constructor));
+
+            // Verify asyncInfo.catchHandler
+            compilation.VerifyPdb("A+<Main>d__0.MoveNext",
```

Comment:
>compilation.VerifyPdb [](start = 12, length = 21)

Can the `[ConditionalFact(typeof(WindowsOnly)]` be replaced with an explicit `if (ExecutionConditionUtil.IsWindows) { ... }` here?